tests/test_route.py

```diff
@@ -130,13 +114,11 @@ def test_RouteAddRemoveIpv4Route(self, dvs, testlog):
         time.sleep(1)
 
         # check application database
-        tbl = swsscommon.Table(self.pdb, "ROUTE_TABLE")
-        route_entries = tbl.getKeys()
+        route_entries = self.pdb.get_keys("ROUTE_TABLE")
```

Comment:
Similarly there is this method for deleted keys: https://github.com/Azure/sonic-swss/blob/3052fb5f25fea0ba2f60ad17c294e6da8b9eb70a/tests/dvslib/dvs_database.py#L357

Reply:
Using wait_for_deleted_entry() to wait until the key is deleted.